.github/workflows/release.yml

```diff
@@ -47,8 +47,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.9", "3.10"]
-    needs:
```

Comment:
@khorshuheng, is this change intentional or something that was done temporarily, to allow the testing flow to proceed?

Reply:
This was done to allow testing flow to proceed.